src/contracts/LANDRegistry.js

```diff
@@ -31,7 +49,22 @@ export class LANDRegistry extends Contract {
     switch (data.version.toString()) {
       case '0': {
         const { version, name, description, ipns } = data
-        return [version, name, description, ipns].join(',')
+        if (name.length > MAX_NAME_LENGTH) {
+          throw new Error(
+            `Error: name is too long, max length allowed is ${MAX_NAME_LENGTH} chars`
```

Comment:
The `Error: ` is not really necessary as it'll be repeated on an eventual logging:
![image](https://user-images.githubusercontent.com/692077/35908725-c7c0b792-0bf1-11e8-9c2b-059590c992ef.png)